### ISSUE TITLE
AppSync - VTL - String Methods Not Supported

### ISSUE THREAD

**Issue by ngouass** [localstack/localstack#2718]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description
Trying to use String methods returns null 
`#set($formattedShopName = "${ctx.args.shop.shopName.trim().toLowerCase().replace(' ','-')}")`

## Expected behavior
`For shopName = " Apple Store " // FormattedShopName = "apple-store"`

## Actual behavior
`For shopName = " Apple Store " // FormattedShopName = null`

# Steps to reproduce

## Command used to start LocalStack
```
version: '2.1'

services:
  localstack:
    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
    image: localstack/localstack
    ports:
      - "4566-4700:4566-4700"
      - "443:443"
      - "${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
    environment:
      - SERVICES=serverless,cloudformation,dynamodb,iam,s3,appsync,edge
      - DEBUG=1
      - DATA_DIR=${DATA_DIR- }
      - PORT_WEB_UI=${PORT_WEB_UI- }
      - LAMBDA_EXECUTOR=${LAMBDA_EXECUTOR- }
      - KINESIS_ERROR_PROBABILITY=${KINESIS_ERROR_PROBABILITY- }
      - DOCKER_HOST=unix:///var/run/docker.sock
      - HOST_TMP_FOLDER=${TMPDIR}
      - LOCALSTACK_API_KEY=${LOCALSTACK_API_KEY}
    volumes:
      - "${TMPDIR:-/tmp/localstack}:/tmp/localstack"
      - "/var/run/docker.sock:/var/run/docker.sock"
```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

...

**Comment by whummer** [comment]
Thanks for reporting @ngouass . This should be fixed in #2719 - can you please give it another try with the latest version of the Docker image? Please report here if the problem persists. Thanks

**Comment by ngouass** [comment]
Thanks for the quick fix @whummer, it works WELL now !
